client/src/rpc_response.rs

```diff
@@ -359,6 +359,17 @@ pub struct RpcPerfSample {
     pub sample_period_secs: u16,
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RpcEpochReward {
```

Comment:
Maybe this name instead?
```suggestion
pub struct RpcInflationReward {
```